src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -14749,7 +14798,16 @@ Label900: ;
 		// For debugging only
 		//CALL  ShowWarningError('LookupSItoIP in: ' // TRIM(stringInWithSI) // ' out: ' // TRIM(stringOutWithIP))
 		//IF (foundConv .NE. 0) CALL  ShowWarningError('   Hint ' // TRIM(UnitConv(foundConv)%hint) // IntToStr(foundConv) )
+
 		unitConvIndex = selectedConv;
+
+		// Add warning if units not found. Exclude stirngs without square brackets, or Rating metrics that are inheirently IP (e.g. SEER, IEER, EER, IPLV)
+		if (unitConvIndex == 0 && ! noBrackets && unitSIOnly != "BTU/W-H") {
+			ShowWarningError("Unable to find a unit conversion from " + stringInWithSI + " into IP units");
+			ShowContinueError("Applying default conversion factor of 1.0");
+			ShowContinueError("Please report to https://github.com/NREL/EnergyPlus/issues");
```

Comment:
I think it's nice to know how the conversion is being handled in the event of an unsupported unit (previously it was multiply by zero).

I suppose the real danger of pointing to GitHub issue tracker is that inexperienced users may misinterpret this as a support service. I'm stepping back on that proposal. I think we would like users to issue a bug report if they come across this, but maybe we let them come to that conclusion.